site/src/sphinx/server-docservice.rst

```diff
@@ -188,3 +190,31 @@ For gRPC:
               }
           }
       }
+
+  Using maven:
+
+  .. code-block:: xml
+
+    <plugin>
```

Comment:
How about adding some comment block to the full manual of `protobuf-maven-plugin`? e.g.

```xml
<!-- See https://www.xolstice.org/protobuf-maven-plugin/usage.html for more information. -->
```